.github/workflows/jsdoc.yml

```diff
@@ -68,24 +68,24 @@ jobs:
       #   id: check_links
       #   uses: modernweb-dev/check-html-links-action@v1
       #   with:
-      #     doc-folder: 'packages/mathtype-html-integration-devkit/out/'
+      #     doc-folder: 'packages/devkit/out/'
 
       # 07. Create a report with any broken link and/or image found.
       - name: Generate the broken links report
         run: |
-          npm run build-jsdoc-validation |& tee packages/mathtype-html-integration-devkit/out/report.txt 
-          cat packages/mathtype-html-integration-devkit/out/report.txt
+          yarn run build-jsdoc-validation |& tee packages/devkit/out/report.txt
```

Comment:
Any reason to use `yarn` commands instead of `nx` ones

Reply:
There is no need of interacting with the other packages and we are not caching this (so far), so might as well directly use yarn IMO.